Sources/Opus/Opus.Decoder.swift

```diff
@@ -73,30 +109,58 @@ public extension Opus.Decoder {
 // MARK: Private decode methods
 
 extension Opus.Decoder {
-	private func decode(_ input: UnsafeBufferPointer<UInt8>, to output: UnsafeMutableBufferPointer<Int16>) throws -> Int {
-		let decodedCount = opus_decode(
-			decoder,
-			input.baseAddress!,
-			Int32(input.count),
-			output.baseAddress!,
-			Int32(output.count),
-			0
-		)
+	private func decode(_ input: UnsafeBufferPointer<UInt8>,
+	                    to output: UnsafeMutableBufferPointer<Int16>,
+	                    packetSize: Int32?) throws -> Int
+	{
+		var decodedCount: Int32 = 0
+		if let size = packetSize {
+			decodedCount = opus_custom_decode(
+				decoder,
+				input.isEmpty ? nil : input.baseAddress,
+				size,
+				output.baseAddress!,
+				Int32(output.count)
+			)
+		} else {
+			decodedCount = opus_decode(
+				decoder,
+				input.isEmpty ? nil : input.baseAddress,
```

Comment:
This change means that if you pass an empty data packet, opus decoder gets the nil it needs to signify a dropped packet.